htsget_server/authz.py

```diff
@@ -15,15 +15,16 @@ def is_authed(id_, request):
         print("WARNING: TEST MODE, AUTHORIZATION IS DISABLED")
         app.logger.warning("WARNING: TEST MODE, AUTHORIZATION IS DISABLED")
         return 200 # no auth
-    if is_site_admin(request):
-        return 200
     if "Authorization" in request.headers:
         authed_datasets = get_authorized_datasets(request)
```

Comment:
I don't think we need to get the list of authed_datasets here. I think we can simply use the id to get the dataset name and pass that with the other request info to is_permissible.